README.md

```diff
@@ -159,8 +155,13 @@ Always confirm your logs are arriving at your Logz.io account.
 
 
 ## Change log
+- **0.1.0**:
+    - Upgrade to use Go modules (Thanks @camal-cakar-gcx)
+    - Update to fluent-bit `1.8.3` in docker image.
+    - Update to latest fluent-bit-go.
+    - Upgrade to `gopkg.in/yaml.v2 v2.4.0`.
```

Comment:
in the go.su, you are using gopkg.in/yaml.v3 v3.0.0

Reply:
fixed